### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by leplaekevin**
added a toc to the readme , i hope this is how u want it.

### CODE REVIEW

**Comment by leplaekevin** [comment]
Sure i'll have a look at it tonight.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,54 @@
...
+##### Developed with
+ 
+* ASP.NET Core 2.0 / C# 7
+* Uses [JSON Flat File Data Store](https://github.com/ttu/json-flatfile-datastore) to store data
+* Can be used without .NET
+  * Docker [#](#docker) 
+  * Self-contained Application [#](#self-contained-application)
+
+
 # Table of contents
```
Comment (ttu) [inline README.md]: Change this to `## Table of contents`. Then this is good 👍

Comment (leplaekevin) [reply README.md]: ok, is done :)

**Comment by ttu** [comment]
Thanks again! Merged and closes #10

**Pull request merged**